### PR TITLE
Adding Python Random Generator app

### PR DESCRIPTION
**Pull request by mcornaton**

### CODE REVIEW

File: iExecBlockchainComputing/PythonRandomGenerator/iexec.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "tag": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "trust": "0",
+    "callback": "0x0000000000000000000000000000000000000000"
+  },
+  "app": {
+    "owner": "0xd8DA71D0302499ACB098Ef815234867897C034fE",
+    "name": "PythonRandomGenerator",
+    "price": 1,
+    "type": "DOCKER",
+    "multiaddr": "docker.io/mcornaton/python-random-generator:latest",
```
Comment (PierreJeanjacquot) [inline iExecBlockchainComputing/PythonRandomGenerator/iexec.json:23]: do not use latest tag

File: iExecBlockchainComputing/PythonRandomGenerator/iexec.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  "social": {
+    "website": "?",
+    "github": "?"
+  },
+  "logo": "logo.png",
+  "buyConf": {
+    "params": {
+      "iexec_args": ""
+    },
+    "tag": "0x0000000000000000000000000000000000000000000000000000000000000000",
```
Comment (PierreJeanjacquot) [inline iExecBlockchainComputing/PythonRandomGenerator/iexec.json:14]: if expected to run in tee enable tee tag "0x0000000000000000000000000000000000000000000000000000000000000001" if non tee this is fine

File: iExecBlockchainComputing/PythonRandomGenerator/iexec.json (hunk 1 above)
Comment (PierreJeanjacquot) [reply iExecBlockchainComputing/PythonRandomGenerator/iexec.json:23]: please push image on iexechub docker hub account

File: iExecBlockchainComputing/PythonRandomGenerator/iexec.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    "callback": "0x0000000000000000000000000000000000000000"
+  },
+  "app": {
+    "owner": "0xd8DA71D0302499ACB098Ef815234867897C034fE",
+    "name": "PythonRandomGenerator",
+    "price": 1,
+    "type": "DOCKER",
+    "multiaddr": "docker.io/mcornaton/python-random-generator:latest",
+    "checksum": "0x15bed530c76f1f3b05b2db8d44c417128b8934899bc85804a655a01b441bfa78",
+    "mrenclave": "6660bd72614d91b25308bc6f20cbb00c0d3da8a60ccd5c72c0f03b5fe18f1d3f"
```
Comment (PierreJeanjacquot) [inline iExecBlockchainComputing/PythonRandomGenerator/iexec.json:25]: must be a standard V6 mrenclave (object or undefined)

File: iExecBlockchainComputing/PythonRandomGenerator/src/app.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import json
```
Comment (PierreJeanjacquot) [inline iExecBlockchainComputing/PythonRandomGenerator/src/app.py:1]: sources are not required, this repo is not intended to store the source code.

File: iExecBlockchainComputing/PythonRandomGenerator/deployed.json [hunk 5]
```diff
@@ -0,0 +1,5 @@
+{
+  "app": {
+    "133": "0xa38e7b89d40503Bdf9dd7B87380886F01a5C8364"
```
Comment (PierreJeanjacquot) [inline iExecBlockchainComputing/PythonRandomGenerator/deployed.json:3]: the app must be published on the marketplace see `iexec orderbook app <address> [options]` result

**Review by PierreJeanjacquot — CHANGES_REQUESTED** [review]
see comments

**Comment by mcornaton** [comment]
I close this pull request to submit a PR to v6 instead of v5. See there: https://github.com/iExecBlockchainComputing/iexec-dapps-registry/pull/124

**Pull request closed without merging**